### DISCUSSION TITLE
AsyncStorage as CustomAuthStore for React Native Development

### DISCUSSION THREAD

**Discussion by gaganbiswas** [pocketbase/pocketbase#2392] [Q&A] [answered]
I am currently using pocketbase to create an react native app using expo. But, I am having trouble customizing the BaseAuthStore to support Async Storage. Is there any solution to use AsyncStorage as the CustomAuthStore instead of browser local storage in react native development. Just like in firebase we can pass in the custom storage where we want to store our auth details.

**Comment by ganigeorgiev** [comment, marked as answer]
You can check https://github.com/pocketbase/pocketbase/discussions/1856.

**Reply by gaganbiswas** [reply]
Thank you very much!
